ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -360,6 +364,45 @@ def ocs_install_verification(
         check_fips_enabled()
 
 
+def osd_encryption_verification():
+    """
+    osd_encryption_verification
```

Comment:
This can be changed to something more readable?
Verify if OSD encryption at rest if successfully deployed on OCS

Or something like this